client/index.dev.jsx

```diff
@@ -31,7 +33,7 @@ const render = () => {
                     position='top-right'
                     transitionIn='fadeIn'
                     transitionOut='fadeOut' />
-                <Application />
+                <React.Suspense fallback='loading...'><Application /></React.Suspense>
```

Comment:
I'm not sure this is necessary because we don't use lazy loading

Reply:
I agree. I tried XHR lazy loading and it did not work (some react hook ordering that I have to troubleshoot). Adding XHR is the plan for the future to avoid loading 11 translation files. I do not see a problem keeping it here just for safety, if you don't mind. Removing is also an easy fix :)

Reply:
Webpack will put the json files in the bundle, so we won't be loading it dynamically.  Let's remove it for now and if we decide to go fully dynamic in the future we can reintroduce it

Reply:
Ok, puhsing the changes.